isminet/models/validators.py

```
@@ -1,6 +1,6 @@
 """Common validators and fields for UniFi Network API models."""
 
-from typing import Optional, List
+from typing import Optional, List, Callable, Any
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)